rust/operator-binary/src/container.rs

```diff
@@ -787,6 +820,23 @@ wait_for_termination $!
                         .build(),
                 );
 
+                if *role == HdfsRole::DataNode {
+                    if let Some(listener_class) = merged_config.listener_class() {
+                        volumes.push(
+                            VolumeBuilder::new(LISTENER_VOLUME_NAME)
+                                .ephemeral(
+                                    ListenerOperatorVolumeSourceBuilder::new(
+                                        &ListenerReference::ListenerClass(
+                                            listener_class.to_string(),
+                                        ),
+                                    )
+                                    .build(),
```

Comment:
After upgrading to the latest operator-rs version:

```suggestion
                                    .build_ephemeral(),
```